designer/server/src/main/scala/pl/touk/nussknacker/ui/process/deployment/DeploymentServiceImpl.scala

```diff
@@ -627,4 +627,4 @@ class DeploymentServiceImpl(
 private class FragmentStateException extends BadRequestError("Fragment doesn't have state.")
 
 private case class CustomActionNonExisting(req: CustomActionRequest)
```

Comment:
Here you still use `.value` instead of just `name`. Also, let's `CustomActionNonExisting` take the name instead of the whole request

Reply:
Fixed